### PR TITLE
Always center text vertically in snapshot tests

### PR DESCRIPTION
**Pull request by swankjesse**
Previously text was centered vertically on iOS and top-aligned on Android Views and ComposeUI.

This makes our snapshot tests more similar across platforms.

### CODE REVIEW

File: redwood-layout-composeui/src/test/snapshots/images/app.cash.redwood.layout.composeui_ComposeUiBoxTest_testChildren_Fill_Center_Stretch[LTR].png
Comment (swankjesse) [inline redwood-layout-composeui/src/test/snapshots/images/app.cash.redwood.layout.composeui_ComposeUiBoxTest_testChildren_Fill_Center_Stretch[LTR].png:1]: Now matches iOS:
https://github.com/cashapp/redwood/blob/trunk/redwood-layout-uiview/RedwoodLayoutUIViewTests/__Snapshots__/UIViewBoxTestHost/testChildren_Fill_Center_Stretch.1.png

File: redwood-layout-composeui/src/test/snapshots/images/app.cash.redwood.layout.composeui_ComposeUiFlexContainerTest_testFlexDistributesWeightUnequally[LTR].png
Comment (swankjesse) [inline redwood-layout-composeui/src/test/snapshots/images/app.cash.redwood.layout.composeui_ComposeUiFlexContainerTest_testFlexDistributesWeightUnequally[LTR].png:1]: Another example where we’re closely matching iOS now:
https://github.com/cashapp/redwood/blob/trunk/redwood-layout-uiview/RedwoodLayoutUIViewTests/__Snapshots__/UIViewFlexContainerTestHost/testFlexDistributesWeightUnequally.1.png

File: redwood-layout-view/src/test/snapshots/images/app.cash.redwood.layout.view_ViewBoxTest_testChildren_Fill_Center_Stretch[LTR].png
Comment (swankjesse) [inline redwood-layout-view/src/test/snapshots/images/app.cash.redwood.layout.view_ViewBoxTest_testChildren_Fill_Center_Stretch[LTR].png:1]: Above I called out how Compose tracks the change. Here we can see Views track it too.

File: redwood-layout-view/src/test/snapshots/images/app.cash.redwood.layout.view_ViewFlexContainerTest_testFlexDistributesWeightUnequally[LTR].png
Comment (swankjesse) [inline redwood-layout-view/src/test/snapshots/images/app.cash.redwood.layout.view_ViewFlexContainerTest_testFlexDistributesWeightUnequally[LTR].png:1]: Another sample to see how we’re now matching on 3 platforms.

**Review by JakeWharton — APPROVED** [review]

**Pull request merged**